### DISCUSSION TITLE
PyAOS, Pythia and AMS Python Symposium

### DISCUSSION THREAD

**Discussion by DamienIrving** [ProjectPythia/projectpythia.github.io#210] [Ideas]
At the AMS Annual Meeting this week there was [some discussion](https://twitter.com/amspython/status/1486330884907282438) about the PyAOS, Pythia and AMS Python Symposium initiatives joining forces where possible, given the obvious overlap in audience and aims. I'd love to see this happen, so I thought I'd start a thread on the topic. 

From the PyAOS perspective, there's basically six things up at the [PyAOS website](https://pyaos.github.io/):
1. Mailing list 
2. Summary of the PyAOS stack
3. Software installation guide
4. Package index
5. List of training resources
6. Census

In terms of the PyAOS mailing list, see #208 for a discussion about Pythia/UCAR hosting the list.

The PyAOS stack summary and installation guide content could easily be transitioned/incorporated into Pythia Foundations - see https://github.com/ProjectPythia/pythia-foundations/issues/220.

The PyAOS package index and list of training resources overlaps with the Pythia Resource Gallery, so I raised the idea of a Pythia package index last year (see #129). The Pythia-Education group indicated they weren't yet ready to create and maintain this sort of authoritative index, but were happy to keep the idea/discussion open going forward.

At the end of my [PyAOS Census talk](https://ams.confex.com/ams/102ANNUAL/meetingapp.cgi/Paper/388788) at AMS 2022 I raised the idea of a larger annual survey (using the learnings from the first PyAOS census as a guide), a bit like the [Stack Overflow Developer Survey](https://insights.stackoverflow.com/survey) but for the PyAOS / Pythia community. I'd be very happy to explore this idea further.

**Comment by scollis** [comment]
Mainly entering a comment so I get updates. 
For one I think the more "pulling in the same direction" the better
for two, the powers that be like to identify "The one thing I can support".. Competition is good, but coalescing around the obvious solution is better

**Comment by ktyle** [comment]
@DamienIrving I'm going to email you so we (Pythia Outreach Working Group members) can schedule a video chat (at a time that works for you in your time zone 😄 )

**Comment by scollis** [comment]
Can you include me please

**Reply by ktyle** [reply]
@scollis will do!
